src/com/org/bean/Ticket.java

```diff
@@ -0,0 +1,46 @@
+package com.org.bean;
+
+public class Ticket {
+	private int ticket_no;
```

Comment:
Use camel case for members of the class. Eg: ticketNumber
```suggestion
	private int ticket_no;
```